### PR TITLE
Refactored printStats, becoming serveStats!

### LINKED ISSUE #47: [REFACTOR] Rename the file printStats.go and the functions inside it to more meaningful names
**Is your feature request related to a problem? Please describe.**
The file `printStats.go` and the functions inside it such as `PrintMemRates` and `PrintCPURates` were named that way when development started and formats of outputs were being decided. It makes more sense to rename the file and the functions inside it to something more apt such as `ServeMemRates`, `ServeCPURates` etc. 

If changed, the corresponding function calls in other source code files must also be changed accordingly :smile:

### PR DESCRIPTION
**Pull request by mattharwood**
# Description

As per issue #47, I have refactored printStats.go, which is now serveStats.go. Functions have been renamed, and mentions of the functions throughout the project updated. I've also added comments to exported functions in the serveStats.go file.

Fixes #47 

## Type of change

Please delete options that are not relevant.

- [ x ] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ x ] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [ x ] I have performed a self-review of my own code (if applicable)
- [ x ] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [ x ] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [ x  ] I have made corresponding changes to the documentation (if applicable)
- [ x ] My changes generate no new warnings 
- [ x ] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Review by MadhavJivrajani — APPROVED** [review]
Hi!
Thanks for your contribution! It looks good to me, I'll wait for another maintainer to approve the changes before merging :smile:

**Comment by mattharwood** [comment]
> 
> 
> Hi!
> Thanks for your contribution! It looks good to me, I'll wait for another maintainer to approve the changes before merging 😄

Thanks Madhav! It's actually my first contribution on Github ever 😁 Much appreciated!

**Review by metonymic-smokey — APPROVED** [review]

**Pull request merged**
